### DISCUSSION TITLE
Errore e blocco Home assistant

### DISCUSSION THREAD

**Discussion by Simone8787** [indomus/forum#233] [HUB: Home Assistant]
Salve. ho istallato da poco home assistant su un raspberry per la gestione domotica della mia casa. saltuariamente, il sistema si blocca. devo spegnere e riaccendere il raspberry, e nel report di home assistant appare questa scritta

WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly

cosa devo fare?

**Comment by snakuzzo** [comment]
Niente...dovrebbe risolvere da se

**Reply by Simone8787** [reply]
eh...solo che non lo fa.
ora provo a vedere se il problema persiste. intanto grazie per la risposta

**Reply by snakuzzo** [reply]
Se persiste e non sei interessato allo storico dei dati...cancella il file del DB

**Reply by Simone8787** [reply]
come si fa? scusa ma sono un po tardo=)

**Reply by sky7176** [reply]
> 
> 
> come si fa? scusa ma sono un po tardo=)

Ciao,
devi entrare nella cartella "config" e cancellare il file "home-assistant_v2.db".

Per entrare nella cartella potresti installare "Samba Share"

Giuseppe
